src/carousel/carousel.scss

```diff
@@ -26,24 +24,28 @@
     flex-shrink: 0;
     box-sizing: border-box;
   }
+
+  ~ .iui-carousel-navigation {
```

Comment:
Why is the slider controlling the styles of the nav? Why not leave it in the navigation mixin? This is a bad pattern which kinda breaks the whole point of using mixins by tightly coupling the styles to the class names.

Also do we even need this border? Looks really weird to me

Reply:
The thought was if someone only used the slides or only the navigation, the divider line wouldn't be present.  When would this happen, I have no idea.  I'll move this out of the slider mixin and put it back in the navigation mixin.
![Screen Shot 2022-01-26 at 5 11 05 PM](https://user-images.githubusercontent.com/849817/151255538-079a64fc-c335-422e-baac-02b7dfb38630.png)